File: csharp/module1/solutions/solution1.cs
public double? CalculateTip(double billAmount, double tipPercentage) {
    if (billAmount < 0 || tipPercentage < 0) {
        Console.Error.WriteLine("Error");
        return null;
    }

    double tipAmount = billAmount * (tipPercentage / 100);
    return Math.Round(tipAmount, 2);
}

File: javascript/module1/solutions/solution1.js
function calculateTip(billAmount, tipPercentage) {
  if (billAmount < 0 || tipPercentage < 0) {
    console.error("Error");
    return null;
  }

  let tipAmount = billAmount * (tipPercentage / 100);
  return tipAmount.toFixed(2);
}
